### DISCUSSION TITLE
ShopifyProduct - hide from menu by default?

### DISCUSSION THREAD

**Discussion by jsirish** [dynamic/silverstripe-shopify#53] [Q&A]
I realized we are not hiding ShopifyProduct from navigation by default. Do we want to set the default to false?

With the current importer, products are in the nav and it's blowing up the layout.

**Comment by mak001** [comment]
I think they should be hidden from the navigation by default.
However, i would make it a config option, that way if its a smaller retail site that only has a couple products they have the option to show by default.
